images/style.css

```diff
@@ -0,0 +1,212 @@
+* {
+    margin: 0;
+    padding: 0;
+}
+
+html,
+body {
+    height: 100vh;
+    font-family: sans-serif;
+}
+
+.page {
+    display: grid;
+    grid-template:
+        'left-side-bar main-area right-side-bar' auto /
+        200px 1fr 300px;
+    border: 3px dotted blue;
+}
+
+
+.left-side-bar {
+    display: grid;
+    grid-area: left-side-bar;
+    grid-template:
+        'left-menu' 630px
+        'tweet-button' 1fr
+        'name' 1fr /
+        200px;
+    border: 3px dotted orange;
+}
+
+.left-menu {
+    display: grid;
+    grid-area: left-menu;
+    grid-template-rows: repeat (9, 70px);
+    border: 3px dotted black;
+
+}
+
+.twitter-logo {
+    display: grid;
+    /* align-items: center;
+    justify-content: center; */
+    position: relative;
+    top: 10px;
+    left: 30px;
+    /* border: 3px dotted red; */
+    
+}
+
+img {
+    width: 30px;
+    height:auto
+}
+
+
+.blue-home {
+    display: grid;
+    color: #71c9f8;
+    font-size: larger;
+    text-align: center; 
+    position: relative;
+    top: 10px
+}
+
+.hashtag {
+    display: grid;
+    color: black;
+    font-size: larger;
+    text-align: center; 
+    position: relative;
+    top: 10px
+}
+
+.icon {
+    display: grid;
+    position: relative;
+    top: 10px;
+    left: 30px;
+    grid-template-columns: 50px 50px;
+    /* border: 3px dotted black; */
+}
+
+.tweet-button {
+    display: grid;
+    grid-area: tweet-button;
+    align-items: center;
+    justify-content: center;
+    border: 3px dotted red;
+   
+} 
+
+.social-link {
+    border: none;
+    padding: 10px 50px;
+    border-radius: 20px;
+
+    text-decoration: none;
+    font-size: 10px;
+    font-weight: bold;
+    color: white;
+}
+
+.Twitter {
+    background-color: #71c9f8;
+    color: white;
+
+}
+
+.name {
+    display: grid;
+    grid-area: name;
+    align-items: center;
+    justify-content: center;
+    border: 3px dotted purple;
+}
+
+.main-area {
+    display: grid;
+    grid-area: main-area;
+    grid-template:
+        'home' 70px
+        'post' 140px
+        'feed' 1fr /
+        1fr;
+    min-width: 500px;
+    border: 3px dotted orange;
+}
+
+.home {
+    display: grid;
+    grid-area: home;
+    align-content: center;
+    margin-left: 20px;
+    /* border: 3px dotted purple; */
+}
+
+.post {
+    display: grid;
+    grid-area: post;
+    position: relative;
+    left: 60px;
+    top: 20px;
+    /* border: 3px dotted purple; */
+    font-size: larger;
+    color: grey;
+}
+
+.feed {
+    display: grid;
```

Comment:
display: grid; is repeated a lot, you could have a section where all of the classes that are grid are in 1 list and if you add elements which you want to be a grid, just to add it to that list.
or you could do what i mentioned in class just have 1 class for example .grid { display: grid;} then add that grid class in your html to which ever elements you want to be a grid